msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -14,7 +14,7 @@ The Live Share SDK can be added to your meeting extension's `sidePanel` and `mee
 > [!Note]
 > Currently, only scheduled meetings are supported, and all participants must be on the meeting calendar. Meeting types such as, one-on-one calls, group calls, and meet now are currently not supported.
 
-:::image type="content" source="../assets/images/teams-live-share/Teams-live-share-dashboard.png" alt-text="Teams Live Share":::
+:::image type="content" source="../assets/images/teams-live-share/Teams-live-share-dashboard.png" alt-text="The screenshot displays Teams live share features.":::
```

Comment:
* I cannot seem to tell what features of live share are shown in the screenshot? What is the screenshot trying to show is not clear? May be work with the original author of the article to update it.
* Downsample the image - it seems to be quite bulky in size.
* Change casing in the image file name here.

Reply:
resolved